routes/orderProducts.js

```diff
@@ -0,0 +1,48 @@
+const orderProductsRouter = require("express").Router();
+const { getOrderProductsById, addProductToOrder, updateOrderProduct, destroyOrderProduct} = require("../db")
+// const { isLoggedIn, isAdmin } = require("./util") need?
```

Comment:
we should use this for creating an order product, updating and order_product, and deleting and order_product

Reply:
updated and resolved.